dev/test-cluster/spark-env.sh

```diff
@@ -0,0 +1,13 @@
+echo $CMPLR_ROOT
+echo $LIBRARY_PATH
+echo $DAALROOT
+echo $CCL_ROOT
+echo $LD_LIBRARY_PATH
+echo $GITHUB_WORKSPACE
+echo $FI_PROVIDER_PATH
+echo $CCL_ATL_TRANSPORT
+export FI_PROVIDER_PATH=$FI_PROVIDER_PATH:/opt/intel/oneapi/mpi/2021.5.1/libfabric/lib/prov:/usr/lib64/libfabric
+ls /opt/intel/oneapi/mpi/2021.5.1/libfabric/lib/prov
+ldd $GITHUB_WORKSPACE/mllib-dal/src/main/resources/lib/libMLlibDAL.so
+export LIBRARY_PATH=$LIBRARY_PATH:/opt/intel/oneapi/tbb/2021.5.1/env/../lib/intel64/gcc4.8:/opt/intel/oneapi/mpi/2021.5.1//libfabric/lib:/opt/intel/oneapi/mpi/2021.5.1//lib/release:/opt/intel/oneapi/mpi/2021.5.1//lib:/opt/intel/oneapi/dal/2021.5.3/lib/intel64:/opt/intel/oneapi/compiler/2022.0.2/linux/compiler/lib/intel64_lin:/opt/intel/oneapi/compiler/2022.0.2/linux/lib:/opt/intel/oneapi/ccl/2021.5.1/lib/cpu
```

Comment:
why hardcode this, not use setvars for each component instead?

Reply:
Just debug log, it will be removed after testing successfully.